commcare_export/env.py

```diff
@@ -242,7 +250,16 @@ def str2date(val):
     import dateutil.parser as parser
     if not val:
         return None
-    date = parser.parse(val)
+
+    if not isinstance(val, six.string_types):
+        val = str(val)
+
+    val = val.encode('utf8')
```

Comment:
Are you sure `val` can't already be a UTF-8 bytestring? If it is, this will throw an exception (UnicodeDecodeError in Py2 and AttributeError in Py3).

If you know `val` will be Unicode or ASCII then this is fine. But if it could be anything, you should probably check before calling `encode()`, maybe like
```
if not isinstance(val, bytes):  # or if isinstance(val, six.text_type):
    val = val.encode('utf8')
```